Helm/opensearch/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.0.0
+version: 1.1.0
```

Comment:
I would prefer not to change the version for now because there are plans to move helm charts to a new repository.

Reply:
I see. Do you plan to keep the two charts separate or merge them like the Opendistro one?

Reply:
I wanted to understand if the community is comfortable with a monolithic chart or separate charts. If monolithic suits we can merge them.

Reply:
@TheAlgo To be clear, we should not have a monolithic chart. We should have all charts available independently in the same repo AND a monolithic chart using all these individual charts as dependencies. 

I think any thing else would be detrimental to the overall user experience.

Reply:
@DandyDeveloper I meant the same we can have separate ones as well as a merged ones for people to use according to their use cases.

Reply:
Is this issue resolved? @TheAlgo @belfhi?
I dont see the version change anymore just need confirmation from you guys.
If so, please resolve this conversation. Thanks.

Reply:
Looks like it has been reverted back. Everything else looks good to me.